src/overlays/actors/ovl_En_Wonder_Talk2/z_en_wonder_talk2.h

```diff
@@ -6,9 +6,21 @@
 
 struct EnWonderTalk2;
 
+typedef void (*EnWonderTalk2Func)(struct EnWonderTalk2*, GlobalContext*);
+
 typedef struct EnWonderTalk2 {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x24];
+    /* 0x014C */ EnWonderTalk2Func actionFunc;
+    /* 0x0150 */ s16 unk_150; // specified message type
+    /* 0x0152 */ s16 switchFlag; // save information
```

Comment:
This comment is probably not necessary since the member is named.
```suggestion
    /* 0x0152 */ s16 switchFlag;
```